extensions/opentelemetry-exporter-jaeger/runtime/src/main/java/io/quarkus/opentelemetry/exporter/jaeger/JaegerRecorder.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.opentelemetry.exporter.jaeger;
+
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.spi.CDI;
+
+import io.opentelemetry.exporter.jaeger.JaegerGrpcSpanExporter;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.quarkus.runtime.annotations.Recorder;
+
+@Recorder
+public class JaegerRecorder {
+    public void installBatchSpanProcessorForJaeger(JaegerExporterConfig.JaegerExporterBuildConfig buildConfig,
+            JaegerExporterConfig.JaegerExporterRuntimeConfig runtimeConfig) {
+
+        // Only create the JaegerGrpcSpanExporter if an endpoint was set in runtime config
```

Comment:
It does default to localhost, but it can also be reset to empty as well.

Whether to set the default to empty depends on whether that means a worse developer experience for "local" usage?

Reply:
On localhost, one will always set it to null.
And I'm pretty sure it's the case for OpenTracing (the documentation say: by default use a NoopTracer)

Reply:
NoopTracer is a bit different than an empty endpoint. There's a significant difference between OpenTracing and OpenTelemetry.

OpenTracing was an API that without a client library, such as Jaeger, would utilize a NoopTracer by default.

OpenTelemetry is a combined API and implementation that includes the libraries to create traces/spans. The only bit that may or may not be there is an exporter

Reply:
I think it should default to null.